books/RayTracingTheRestOfYourLife.html

```diff
@@ -99,14 +128,14 @@
                 inside_circle++;
         }
         std::cout << std::fixed << std::setprecision(12);
-        std::cout << "Estimate of Pi = " << 4*double(inside_circle) / N << '\n';
+        std::cout << "Estimate of Pi = " << 4 * double(inside_circle) / N << '\n';
     }
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     [Listing [estpi-1]: <kbd>[pi.cc]</kbd> Estimating π]
 </div>
 
 The answer of $\pi$ found will vary from computer to computer based on the initial random seed.
-On my computer, this gives me the answer `Estimate of Pi = 3.0880000000`
+On my computer, this gives me the answer `Estimate of Pi = 3.143760000000`
```

Comment:
Missing period. Also, how about adding “, with an error of about 0.002167.”?

Reply:
Not sure if you mean to solve for it programmatically, e.g.
```
std::cout << ", with an error of about " << pi - (4.0 * inside_circle) / N << '\n';
```

or if I should just type it in, but I don't think it adds very much, and, worse, having an extension on the digits of pi that are wrong is implying that the answer we're spitting out has 12 digits of precision. 
That our solved value of pi has 12 digits of precision and we can compare it to real pi.
When this a wrong way to look at it.

I expect the student to play with the value of N and see how the precision and accuracy changes along with it.

If you wanted to do this, then I recommend that we force the 
```
        std::cout << std::fixed << std::setprecision(12);
```
to stop at 6 digits of precision (or whatever N amounts to), but we could get to weird round off errors where solved value of pi is exactly pi, which is just wrong.